<launch>
    <group ns="auto_label">
        <group ns="autorally">
            <rosparam param="cam_mat">[852.0367654236857, 852.0736515102927, 626.9338016782257, 500.2665457442116]</rosparam>
            <rosparam param="cam_R">[
                -0.0129984, 0.9999121, 0.00261227,
                0.00704835, -0.0025208, 0.99997198,
                0.99989068, 0.01301645, -0.00701496
                ]</rosparam>
            <rosparam param="cam_translation">[0.06893929, 0.02042668, -0.11304977]</rosparam>
        </group>
        <group ns="autorally3">
            <rosparam param="cam_mat">[858.1572184991888, 856.0929924651027, 649.6505015547086, 529.135158176981]</rosparam>
            <rosparam param="cam_R">[
                0.00551476, 0.9999745, -0.0045373,
                -0.01483231, 0.00461867, 0.99987933,
                0.99987479, -0.00544679, 0.0148574
                ]</rosparam>
            <rosparam param="cam_translation">[ 0.06418514, 0.15496709, -0.13159533]</rosparam>
        </group>
    </group>

    <node name="auto_label" pkg="autorally_vision" type="auto_label" output="screen">
        <!-- label_sim, label_real, run -->
        <param name="mode" value="label_real"/>
        <!-- how far into the self bag file to start labeling -->
        <param name="start_time" value="1493237995.0"/>
        <param name="half_width" value="0.15"/>
        <param name="half_length" value="0.3"/>
        <param name="half_height" value="0.1"/>

        <param name="image_width" value="1280"/>
        <param name="image_height" value="1024"/>

        <!-- the z values flucuates in the state estimator a lot, ignored in sim -->
        <param name="my_z" value="0.07"/>
        <param name="other_z" value="0.22"/>
        <!-- used to load the configuration -->
        <rosparam param="compute_box_names">[autorally, autorally3]</rosparam>
        <!-- label only -->
        <!-- hardware -->
        <!--
        <param name="output_dir" value="/home/jason/Documents/singleShotCV/04-26-14-34/vehicle_detection"/>
        <param name="my_bag_file" value="/home/jason/Documents/singleShotCV/04-26-14-34/gamma_autorally3_2017-04-26-14-34-30.bag"/>
        <rosparam param="other_bag_files">[/home/jason/Documents/singleShotCV/04-26-14-34/beta_autorally_2017-04-26-14-34-28.bag]</rosparam>
        -->
        <!--
        <param name="output_dir" value="/home/jason/Documents/singleShotCV/04-26-16-19/vehicle_detection"/>
        <param name="my_bag_file" value="/home/jason/Documents/singleShotCV/04-26-16-19/gamma_autorally3_2017-04-26-16-19-28.bag"/>
        <rosparam param="other_bag_files">[/home/jason/Documents/singleShotCV/04-26-16-19/beta_autorally_2017-04-26-16-19-37.bag]</rosparam>
        -->
        <!--
        <param name="output_dir" value="/home/jason/Documents/singleShotCV/04-26-16-24/vehicle_detection"/>
        <param name="my_bag_file" value="/home/jason/Documents/singleShotCV/04-26-16-24/gamma_autorally3_2017-04-26-16-24-24.bag"/>
        <rosparam param="other_bag_files">[/home/jason/Documents/singleShotCV/04-26-16-24/beta_autorally_2017-04-26-16-24-21.bag]</rosparam>
        -->
        <param name="output_dir" value="/home/jason/Documents/singleShotCV/04-26-17-37/vehicle_detection"/>
        <param name="my_bag_file" value="/home/jason/Documents/singleShotCV/04-26-17-37/gamma_autorally3_2017-04-26-17-37-16.bag"/>
        <rosparam param="other_bag_files">[/home/jason/Documents/singleShotCV/04-26-17-37/beta_autorally_2017-04-26-17-37-11.bag]</rosparam>
    </node>
</launch>
